content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -97,13 +97,11 @@ $ dvc stage add --name train \
                 --params seed,lr,weight_decay \
                 --checkpoints model.pt \
                 --plots-no-cache predictions.json \
```

Comment:
Do you know if `--plots-no-cache predictions.json` is used at all? Seems like that file isn't meant to be a plot and maybe should be `--outs-no-cache` instead. cc @flippedcoder